code/_compile_options.dm

```diff
@@ -38,7 +38,7 @@
 
 //Update this whenever the byond version is stable so people stop updating to hilariously broken versions
 #define MAX_COMPILER_VERSION 514
-#define MAX_COMPILER_BUILD 1554
+#define MAX_COMPILER_BUILD 1557
```

Comment:
cant it be 1561 ?

Reply:
it can

Reply:
actually nevermind debugger is till kaputt